client/src/pages/Profile.js

```diff
@@ -9,14 +9,39 @@ import { withPageContext } from '../components/pageContext';
 
 import './profile.css';
 
+const styles = {
+  pageContent: {
+    display: 'flex',
+  },
+  headerContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  header: {
+    display: 'inline',
+    margin: '10px',
+    padding: '20px',
+    '&:hover': {
+      cursor: 'pointer',
+      color: '#E4E8EA',
+      textShadow: '1px 1px #B2B1B7',
```

Comment:
In my opinion, if we change color, and not add a shadow it would look better then how it currently is,

Reply:
Removed the text shadow